src/main/java/de/hs_mannheim/informatik/ct/controller/Utilities.java

```diff
@@ -86,4 +89,31 @@ private UriComponentsBuilder createUriBuilder(String scheme, String path){
                     .path(path);
         }
     }
+
+    /**
+     * add a cookie to the http response
+     *
+     * @param response      HTTP Response data
+     * @param cookieName    name of the cookie
+     * @param cookieValue   value of the cookie
+     */
+    public void setCookie(HttpServletResponse response, String cookieName, String cookieValue) {
+        Cookie c = new Cookie(cookieName, cookieValue);
+        c.setMaxAge(60 * 60 * 24 * 365 * 5);
```

Comment:
As we check out everybody at midnight, it might make sense, to set this cookie until midnight only, wouldn't it?

Reply:
I have generally looked at the cookie management and introduced a separate class CookieManager. It's maybe a bit over engineered, but in my opinion a bit cleaner :) the max age of the cookies could be set in the cookie manager for each cookie type individually.